csuntechlab/iSTART
# STR-242 -- FE: Fix Input Comparison Validation

# Description
Changed event handler for input comparison module.
  
# Testing
1. Ensure previous validation functionality is not broken.
2. Check to see if validation occurs on keyup.
3. Try to break functionality + on mobile.
